CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ This changelog keeps track of work items that have been completed and are ready
 
 ### New
 
+- **General**: Add multi-host support to `HTTPScaledObject` ([#552](https://github.com/kedacore/http-add-on/issues/552))
```

Comment:
move it to `### Deprecations` please 🙏

Reply:
I already have it in the `Deprecations` section. However looking at the keda changelog they usually specify the feature add in this section and then also add it to `Deprecations` with a new test.

Here is the deprecations part: https://github.com/kedacore/http-add-on/pull/674/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR41

Should I remove it completely from the `New` section?

Reply:
Any news? @JorTurFer

Reply:
f**k, I didn't see it 🤦 
I think that it's good enough how it is